components/server/src/routes/report.py

```diff
@@ -87,6 +89,29 @@ def export_report_as_pdf(report_uuid: ReportId):
     return response.content
 
 
+@bottle.get("/api/v3/report/<report_uuid>/json")
+def export_report_as_json(database: Database, report_uuid: ReportId):
+    """Return the quality report, including information about other reports needed for move/copy actions."""
```

Comment:
Is this part of the comment correct: "including  information about ..."?

Reply:
Mh, I actually don't think it is. Don't remember why I put it there though.